tst/EnergyPlus/unit/WaterThermalTanks.unit.cc

```diff
@@ -5427,3 +5435,279 @@ TEST_F(EnergyPlusFixture, setBackupElementCapacityTest)
     expectedAnswer = -456.0;
     EXPECT_NEAR(DSup.BackupElementCapacity, expectedAnswer, allowedTolerance);
 }
+
+TEST_F(EnergyPlusFixture, MixedTank_PVT_Per_VolumeSizing_PerSolarCollectorArea)
```

Comment:
New unit test. 

A PlantLoop with a WHMixed on the supply, a BIPVT collector on the demand.

WHMixed was Autosized Volume, Sizing is PerCollectorArea